src/pages/main/Header/FunctionOptions.jsx

```diff
@@ -1,7 +1,16 @@
-import { Box, Typography } from '@mui/material'
+import React from 'react'
```

Comment:
```suggestion
import { useState } from 'react'
```
와 같이 TreeShaking을 적용해주시는 습관을 들이시는게 좋습니다